data/engines.txt

```diff
@@ -419,7 +419,7 @@ outfit "A860 Atomic Thruster"
 	"flare sprite" "effect/atomic flare/huge"
 		"frame rate" 10
 	"flare sound" "atomic huge"
-	description "The A860 atomic thruster is the most powerful engine sold in human space, and is much more compact than any of the alternatives. But, you may need to upgrade your ship's power and cooling systems to handle it."
+	description "The A860 atomic thruster is the most powerful engine sold in human space, and is much more compact than any of the alternatives. However, you may need to upgrade your ship's power and cooling systems to handle it."
```

Comment:
Ah, I didn't get what you were going for by... whelming it up. Personally, I _wouldn't_ increase the emphasis on the energy drain as a disadvantage, because in most scenarios I've encountered it's not that significant a disadvantage. Korath reactors, they deserve serious bigging up of their ridiculous heat because it _really_ matters. Here, if you want to big something up, big up how awesome they are: a fair description shouldn't make it sound like a balanced trade-off, as it isn't. They're just awesome. i.e.

`description "The A860 atomic thruster is the capstone of the Deep Sky's atomic thrusters, and quite rightly: it is by far the most powerful engine sold in human space, utilising technology decades ahead of other engine manufacturers. While you may need to upgrade your power and cooling systems to handle it, it - and the price tag - are generally considered a small cost to pay for the performance."`